### ISSUE TITLE
Getting Test Accuracy of 62.9% instead of 59.56% for the whole dataset

### ISSUE THREAD

**Issue by srinadhu** [WellyZhang/RAVEN#3]
Hello @WellyZhang, 

Thanks for making the code public. I used all the default hyperparameters. The environment is Python 2.7, Pytorch 0.4.1, Torchvision 0.2.0. I stopped training at 186 epochs. The best test accuracy I am able to get is 62.9% more than 59.56%. Were you able to get more accuracy than reported in the paper? Is it possible I made somewhere something wrong? If possible could you provide your logs or best models? Thanks and Regards

![raven_reimplemenatiob](https://user-images.githubusercontent.com/23106101/62521381-9ba1ca00-b840-11e9-9d84-ea8f9334cbc4.png)

**Comment by WellyZhang** [comment]
Hi @srinadhu, 

Thank you for your interest in this work and running the code. 

I do have reports about better or worse performance of the models. I've recently rerun the code and notice that the performance is sensitive to your initialization. Sometimes you get better but sometimes worse. 

We will try to update the performance table after a thorough investigation.

**Comment by srinadhu** [comment]
Thanks, @WellyZhang, roughly by when can I expect an update on the performance table?

**Comment by WellyZhang** [comment]
As I'm busy with some other work right now, I will try to do it after
September.

**Comment by srinadhu** [comment]
Sure, thank you.
